streamduck-core/src/config.rs

```diff
@@ -320,17 +322,28 @@ impl Config {
         let path = self.device_config_path();
         fs::create_dir_all(&path).await.ok();
 
-        for (serial, device) in devices.iter() {
-            let device= device.clone();
+        for (_serial, device) in devices.iter() {
```

Comment:
Probably can just ignore the variable all together
```suggestion
        for (_, device) in devices.iter() {
```